id_pkg/test/test_log_parse.py

```diff
@@ -28,7 +28,7 @@ def test_syslog_file(self):
 
         # Open the syslog file
         # https://docs.python.org/3/tutorial/inputoutput.html
-        with open(os.path.join(self.data_path, fname), encoding='utf-8') as f:
```

Comment:
I don't think you updated your branch to latest.  If you commit this you'll break unit test for anyone using a PC

Reply:
My branch says it is up to date, should I try again now?